mvn-extension/pom.xml

```diff
@@ -52,19 +52,21 @@
       <version>${project.version}</version>
       <scope>runtime</scope>
     </dependency>
-    <!-- -->
 
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>com.github.stefanbirkner</groupId>
       <artifactId>system-rules</artifactId>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
```

Comment:
the version in this dependency is not necessary here - it is specified in the parent pom.